marklogic-data-hub-central/ui/src/pages/Modeling.test.tsx

```diff
@@ -85,6 +86,29 @@ describe("Modeling Page", () => {
     expect(getByLabelText("save-all")).toBeDisabled();
     expect(queryByLabelText('entity-modified-alert')).toBeNull();
   });
+
+  test("Modeling: can not see data if user does not have entity model reader role", async () => {
+    mockPrimaryEntityType.mockResolvedValueOnce({ status: 200, data: getEntityTypes });
+
+    const { queryByText, queryByLabelText } = render(
+      <AuthoritiesContext.Provider value={mockHCUserRolesService}>
```

Comment:
This is ok. Another example is use an authority for ex readRole who doesnt have rights to model tile. If setting all the authority to false is same as a user with hub-central-user role, then this test looks good.
```
const authorityService = new AuthoritiesService();
authorityService.setAuthorities(['readLoad']);
```